src/components/form/adapters/widget-adapter.ts

```diff
@@ -11,6 +11,17 @@ interface WidgetAdapterProps {
     events?: { [key: string]: (event: any) => void };
 }
 
+/**
+ * A widget is a concept in react-jsonschema-form (RJSF).
+ * It represents a HTML tag for the user to enter data, eg. input, select, etc.
+ *
+ * You're using a widget when you want to render a field in a form
+ * in a specific way. For boolean fields you could for instance use a `radio` or
```

Comment:
This wording makes it seem like users can override widgets when they can't. Users custom components are rendered as Fields. We use the widget adapter exclusively so we can use lime-elements instead of the default input/select/checkbox/etc fields that rjsf provides.